frontend/src/components/workspaces/Templates/TemplatesTableRow/TemplatesTableRow.tsx

```diff
@@ -41,15 +41,9 @@ const TemplatesTableRow: FC<ITemplatesTableRowProps> = ({ ...props }) => {
       <div className="w-full flex justify-between py-0">
         <Space size={'middle'}>
           {gui ? (
-            <DesktopOutlined
-              className={'primary-color-fg'}
-              style={{ fontSize: '24px' }}
-            />
+            <DesktopOutlined style={{ fontSize: '24px', color: '#1c7afd' }} />
```

Comment:
Why did you hardcoded the values?

Reply:
It's not the first time we hardcode the `fontSize` css property. We have also hardcoded the `color` property before, in the `WorkspaceGridItem`. We did it for the other icons so we decided to do it also for this one. If u want i can open a PR to refactor all icons.

Reply:
I think we'll need to do it, but now now